open_samus_returns_rando/files/levels/s100_area10.lua

```diff
@@ -265,9 +266,10 @@ function s100_area10.ScheduledQueenRoar()
 end
 s100_area10.fSafeFarPlaneFactor = -1
 function s100_area10.OnSubAreaChange(_ARG_0_, _ARG_1_, _ARG_2_, _ARG_3_, _ARG_4_)
-  if _ARG_2_ == "collision_camera_022" and Scenario.ReadFromBlackboard("firstTimeMetroidHatchlingIntroPlayed", false) then
+  if Blackboard.GetProp("DEFEATED_ENEMIES", "Queen") ~= nil then
     Game.SetSceneGroupEnabledByName("sg_egg01", false)
     Game.SetSceneGroupEnabledByName("sg_egg02", true)
+    Game.EnableEntity("LE_Baby_Hatchling")
```

Comment:
I suspect that this could maybe lead to always reenable it a.k.a you can pick it up a multiple/endless times.
Did you checked that?

Reply:
It does not respawn